# Frame names
frames:
  robot_base: "robot_base"
  odom: "odom"
  camera_optical: "arena_camera_optical"
  world: "world"
  arena_tag: "arena_tag"
  robot_top_tag: "arena_perception/robot_top_tag"
  robot_bottom_tag: "arena_perception/robot_bottom_tag"
  robot_base_arena_perception: 'arena_perception/robot/base_footprint'

# Topics
topics:
  robot_pose: "arena_perception/robot/pose"